### PR TITLE
Update eslint, fix all missed lint rules

### PR DESCRIPTION
**Pull request by iest**
It seems we should all read the aribnb style guide a bit closer... there's a bunch of rules we weren't following that also weren't being picked up by eslint. Now they are.

### CODE REVIEW

File: components/Blocks/CTABlock.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import React, {PropTypes} from 'react';
+import React, { PropTypes } from 'react';
```
Comment (tomgatzgates) [inline components/Blocks/CTABlock.js]: YES!

File: components/DatePicker/DatePicker.js [hunk 2]
```diff
@@ -35,12 +35,14 @@ export default class DatePicker extends Component {
   render() {
     return (
       <div className={s.root}>
-        <DayPicker locale="en-gb"
-                   enableOutsideDays
-                   localeUtils={localeUtils}
-                   renderDay={this.renderDay}
-                   onDayClick={this.props.onChange}
-                   {...this.props}/>
+        <DayPicker
```
Comment (tomgatzgates) [inline components/DatePicker/DatePicker.js]: :+1: Readability

**Comment by iest** [comment]
This also solves prop indentation once and for all:

- If props don't fit on one line, they're always indented with 2 spaces
- Closing tags are always on the same column as the opening

```jsx
<Component props="on one line"/>

<Component
  props="on"
  multiple="lines"
/>

<Component
  props="on"
  multiple="lines"
>
  With children
</Component>
```

File: styleguide/sections/Forms.js [hunk 3]
```diff
@@ -171,16 +174,18 @@ export default class FormSection extends Component {
         Is mothercluckers the best chicken in the universe? <strong>{this.state.toggleVal ? 'Absolutely.' : `I don't know. Probably.`}</strong>
```
Comment (rdjpalmer) [inline styleguide/sections/Forms.js]: `’` on "I don't know" please :)

File: styleguide/sections/Overlays.js [hunk 4]
```diff
@@ -63,34 +63,46 @@ export default class Overlays extends Component {
         <p>
           Very similar to TooltipToggle, but the children prop is the content you want inside the dropdown, and you pass in a trigger to open and close the dropdown.
         </p>
-        <div style={{textAlign: 'center'}}>
-          <Dropdown trigger={
-            <Btn className={m.mbs}>Toggle dropdown <Icon name="triangle"/></Btn>
-          }>
+        <div style={{ textAlign: 'center' }}>
+          <Dropdown trigger=
```
Comment (rdjpalmer) [inline styleguide/sections/Overlays.js]: Shouldn't `trigger=` be on the next line as well?

**Comment by rdjpalmer** [comment]
One apostrophe change and a minor query, but LGTM :+1:

File: styleguide/sections/Overlays.js (hunk 4 above)
Comment (iest) [inline styleguide/sections/Overlays.js]: I did have

```jsx
<Dropdown trigger={
  <Btn className={m.mbs}>Toggle dropdown <Icon name="triangle"/></Btn>
}
>
```
which doesn't look right. Then I tried:

```jsx
<Dropdown trigger={
  <Btn className={m.mbs}>Toggle dropdown <Icon name="triangle"/></Btn>
}>
```
but it breaks the listing rules.

Changing to:
```jsx
<Dropdown
  trigger={
    <Btn className={m.mbs}>Toggle dropdown <Icon name="triangle"/></Btn>
  }
>
```

Comment (rdjpalmer) [inline styleguide/sections/Overlays.js]: Looks good :+1:

**Comment by rdjpalmer** [comment]
:ship:

**Comment by iest** [comment]
Also added https://github.com/PactCoffee/loggins/commit/9ea65ad26ae4f9cc45b8a8c03561272b01a152a7 to hard-set react-slider to a known-working version. In future we should look at moving away from that library or develop our own.

**Comment by rdjpalmer** [comment]
@iest seems that last commit is empty?

+1 on moving away from it, it has [accessibility issues](https://trello.com/c/uTQAizFK/119-slider-component-is-not-controllable-by-keyboard).

**Pull request merged**